src/FSharp.Plotly/Cellcolor.fs

```diff
@@ -0,0 +1,30 @@
+﻿namespace FSharp.Plotly
```

Comment:
Please create a type in StyleParam.fs for this

Reply:
Must be a dynamic object. Moved to Table.fs